### PR TITLE
mock session manager

### PR DESCRIPTION
**Pull request by AmosKorir**
Mock Session manager

### CODE REVIEW

**Review by jumaallan — APPROVED** [review]

**Review by maplonki — APPROVED** [review]

**Comment by jumaallan** [comment]
@AmosKorir 

We can use the shared pref file I setup, and make it an observable using LiveData

**Comment by AmosKorir** [comment]
@jumaallan That is fine.

**Comment by codecakes** [comment]
@AmosKorir @jumaallan canu guys merge it if its done?

**Review by codecakes — COMMENTED** [review]
Use smart commits if you are working on a jira user story: https://support.atlassian.com/bitbucket-cloud/docs/use-smart-commits/

File: app/src/main/java/com/ht117/yukute/sessionmanager/SessionManagerInterface.kt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package com.ht117.yukute.sessionmanager
+
+/**
+ * check if the session is valid
+ */
+interface SessionManagerInterface {
+
+    fun isValidSession(): Boolean
+}
```
Comment (codecakes) [inline app/src/main/java/com/ht117/yukute/sessionmanager/SessionManagerInterface.kt:9]: empty line EOF in all files

**Review by codecakes — CHANGES_REQUESTED** [review]

**Comment by codecakes** [comment]
any updates guys?

**Pull request merged**